frontend/test/lit/test_custom_ops.py

```diff
@@ -37,7 +37,7 @@ def decomposition(self):
         return [qml.PauliRot(self.theta, pauli_word="XX", wires=self.wires)]
 
 
-lightning = qml.device("lightning.qubit", wires=3)
+lightning = qml.device("lightning.kokkos", wires=3)
```

Comment:
It just uses kokkos list of operations because the one from lightning is a frozen set.

Reply:
Catalyst integration tests should be rely on LQ by default so that users can run these tests without having LK installed on their machine. Can we cast the frozenset of LQ ops to a list here?

Reply:
Good point let me change that

Reply:
Updated everywhere